services/api/service.go

```diff
@@ -1209,26 +1375,34 @@ func (api *RelayAPI) handleInternalBuilderStatus(w http.ResponseWriter, req *htt
 		return
 	} else if req.Method == http.MethodPost || req.Method == http.MethodPut || req.Method == http.MethodPatch {
 		args := req.URL.Query()
+		isOptimistic := args.Get("optimistic") == "true"
 		isHighPrio := args.Get("high_prio") == "true"
 		isBlacklisted := args.Get("blacklisted") == "true"
 		api.log.WithFields(logrus.Fields{
 			"builderPubkey": builderPubkey,
+			"isOptimistic":  isOptimistic,
 			"isHighPrio":    isHighPrio,
 			"isBlacklisted": isBlacklisted,
 		}).Info("updating builder status")
-
-		newStatus := datastore.MakeBlockBuilderStatus(isHighPrio, isBlacklisted)
-		err := api.redis.SetBlockBuilderStatus(builderPubkey, newStatus)
+		status := common.LowPrio
```

Comment:
Consider matching the ordering in `func (b BuilderStatus) String() string`.

Reply:
done! also rearranged the logging above to match as well.